install/create-docker-volumes.sh

```diff
@@ -6,6 +6,5 @@ echo "Created $(docker volume create --name=sentry-kafka)."
 echo "Created $(docker volume create --name=sentry-postgres)."
 echo "Created $(docker volume create --name=sentry-redis)."
 echo "Created $(docker volume create --name=sentry-symbolicator)."
-echo "Created $(docker volume create --name=sentry-zookeeper)."
```

Comment:
May wanna add clean up for this external volume on upgrades.

Reply:
+1

Reply:
Can probably add this to `turn-things-off.sh`

Reply:
pushed this up